prow/plugins/verify-owners/verify-owners.go

```diff
@@ -231,16 +231,9 @@ func handle(ghc githubClient, gc git.ClientFactory, roc repoownersClient, log *l
 			log.WithError(err).Error("Error cleaning up repo.")
 		}
 	}()
-	if err := r.CheckoutPullRequest(number); err != nil {
+	if err := r.MergeAndCheckout(pr.Base.Ref, string(github.MergeMerge), pr.Head.SHA); err != nil {
```

Comment:
I think the first argument needs to be `pr.Base.SHA` instead of `pr.Base.Ref`

Reply:
Thank you for the review. I have used `pr.Base.Ref` because `pr.Base.SHA` points to the version of the base branch off which the PR was originally created, thus missing updates to the base branch (e.g. addition of the missing alias through a separate PR). I tested with the specified scenario using `pr.Base.SHA` and it missed the update to base branch.